packages/xo-web/src/xo-app/host/index.js

```diff
@@ -118,6 +118,7 @@ export default class Host extends Component {
 
   loop(host = this.props.host) {
     if (host == null) {
+      clearTimeout(this.timeout)
```

Comment:
FYI: The error will keep popping if the user refresh the page. But, it's not a big deal because we only loose the connection a few seconds.